keyboards/handwired/bluepill/bluepill70/config.h

```diff
@@ -0,0 +1,30 @@
+#ifndef STM70_CONFIG_H_
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?